src/main/java/com/clickhouse/kafka/connect/sink/processing/Processing.java

```diff
@@ -140,14 +140,9 @@ public void doLogic(List<Record> records) throws IOException, ExecutionException
                         stateProvider.setStateRecord(new StateRecord(topic, partition, rangeContainer.getMaxOffset(), rangeContainer.getMinOffset(), State.AFTER_PROCESSING));
                         break;
                     case CONTAINS: // The state contains the given records
-                        LOGGER.warn(String.format("Records seemingly missing compared to prior batch for topic [%s] partition [%s].", topic, partition));
                         // Do nothing - write to dead letter queue
-                        LOGGER.warn("Due to CONTAINS state, sending [{}] records to DLQ for topic: {}, partition: {}, minOffset: {}, maxOffset: {}",
-                                records.size(), topic, partition, rangeContainer.getMinOffset(), rangeContainer.getMaxOffset());
-                        records.forEach( r ->
-                                Utils.sendTODlq(errorReporter, r, new DuplicateException(String.format(record.getTopicAndPartition())))
-                        );
-                        break;
+                        throw new RuntimeException(String.format("State CONTAINS given [%s] records for topic: [%s], partition: [%s], minOffset: [%s], maxOffset: [%s]",
```

Comment:
Will this cause the connector to stop? And then, the user needs to make a manual adjustment to offsets to continue ?